plugin/evm/vm.go

```diff
@@ -351,6 +362,10 @@ func (vm *VM) Initialize(
 	vm.chain = ethChain
```

Comment:
We should port `readLastAccepted` for the `lastAcceptedBytes` block: https://github.com/ava-labs/coreth/blob/9fb7a2025a20f3a5546683571f1066cae932457c/plugin/evm/vm.go#L1708-L1732